src/EpicsClient/EpicsClientMonitor.cpp

```diff
@@ -220,64 +217,60 @@ std::string ChannelRequester::getRequesterName() { return "ChannelRequester"; }
 
 void ChannelRequester::message(std::string const &Message,
                                epics::pvData::MessageType MessageType) {
-  LOG(Sev::Warning, "Message for: {}  msg: {}  msgtype: {}", getRequesterName(),
-      Message, getMessageTypeName(MessageType));
+  Logger->warn("Message for: {}  msg: {}  msgtype: {}", getRequesterName(),
+               Message, getMessageTypeName(MessageType));
 }
 
 void ChannelRequester::channelCreated(epics::pvData::Status const &Status,
                                       Channel::shared_pointer const &Channel) {
   // Seems that channel creation is actually a synchronous operation
   // and that this requester callback is called from the same stack
   // from which the channel creation was initiated.
-  LOG(Sev::Debug, "ChannelRequester::channelCreated:  (int)status.isOK(): {}",
-      (int)Status.isOK());
+  Logger->debug("ChannelRequester::channelCreated:  (int)status.isOK(): {}",
```

Comment:
I think this whole method can be simplified to something like:
`std::string ChannelName = "Unknown";
  // May be possible to get name even if channel not created successfully.
  if (Channel) {
    ChannelName = Channel->getChannelName();
  }

  if (Status.isSuccess()) {
    Logger->debug("ChannelRequester::channelCreated: {}",
        ChannelName);
  } else {
    Logger->error("ChannelRequester::channelCreated:  failure: {}", ChannelName);
  }`

Reply:
@mattclarke surely doing `if (Channel)` is an implicit cast to bool? should it not be `if (Channel != nullptr)`?